packages/client-react-streaming/src/ReadableStreamLink.tsx

```diff
@@ -0,0 +1,137 @@
+import { ApolloLink, Observable } from "@apollo/client/index.js";
+import type { FetchResult } from "@apollo/client/index.js";
+
+export type ReadableStreamLinkEvent =
+  | { type: "next"; value: FetchResult }
+  | { type: "completed" }
+  | { type: "error" };
+
+interface InternalContext {
+  [teeToReadableStreamKey]?: ReadableStreamDefaultController<ReadableStreamLinkEvent>;
+  [readFromReadableStreamKey]?: ReadableStream<ReadableStreamLinkEvent>;
+}
+
+const teeToReadableStreamKey = Symbol.for(
+  "apollo.tee.readableStreamController"
```

Comment:
```suggestion
  "apollo.readableStreamController"
```

Is there anything else we'd consider putting on `apollo.tee.*`? If not, I'd simplify this to a single namespace (same goes for the read key)

Reply:
These are common enough that I see them colliding with something else in the future - it's so close to a web standard type that I would combine it with the method calling it: https://developer.mozilla.org/en-US/docs/Web/API/ReadableStreamDefaultController